### ISSUE TITLE
Should no-unsupported-features disable the prefer-rest-params rule on Node.js v4?

### ISSUE THREAD

**Issue by nfriedly** [mysticatea/eslint-plugin-node#63]
I am trying to use this in combination with eslint-config-google.  My package.json specifies engines.node >= 4.

The google config [enables](https://github.com/google/eslint-config-google/blob/master/index.js#L307) the [prefer-rest-params](http://eslint.org/docs/rules/prefer-rest-params) rule, and that does [work on node v4](http://node.green/#rest-parameters) with `--harmony`, but not otherwise.

My expectation was that this plugin/config would override the google one, but that didn't happen, and I have to override it myself in my `.eslintrc.js` file. 

Is that a bug in this plugin, or just incorrect expectations on my part?

**Comment by mysticatea** [comment]
Thank you for this issue.

`no-unsupported-features` rule has `ignores` option: https://github.com/mysticatea/eslint-plugin-node/blob/master/docs/rules/no-unsupported-features.md#ignores
Please use the option for `--harmony` flag family.

For example:

```js
{
    "rules": {
        "node/no-unsupported-features": ["error", {"ignores": ["restParameters"]}]
    }
}
```

**Comment by mysticatea** [comment]
I might misread this issue.

Rules of ESLint are independent each other by ESLint's Philosophy.
So rules never change the behavior of other rules except few exceptions.
